ibis/backends/duckdb/tests/test_geospatial.py

```diff
@@ -340,3 +340,10 @@ def test_geo_binop_geo_literals(ext_dir, geo_line_lit):
     expr = geo_line_lit.distance(shapely.Point(0, 0))
 
     assert con.execute(expr) == 0
+
+
+def test_cast_wkb_to_geo(con):
+    t = con.table("geo_wkb")
+    geo_expr = t.geometry.cast("geometry").centroid()
+    con.execute(geo_expr)  # check that the we can do a geo op
```

Comment:
This test should verify that the output column is actually a geometry/geography dtype, otherwise it's not actually testing the effect of the cast.

Reply:
The output column will be converted to pandas, I was not able to figure out how to check that was a geometry, but I thought that if I can perform a geospatial operation then that confirm it's a geometry column, otherwise I wouldn't be able too.

Reply:
I think an assertion that the column is a geopandas series is enough. The result isn't an Ibis expression, so e.g., checking `result.type().is_geospatial()` won't work. Something like `assert isinstance(result, gpd.GeoSeries)` should suffice.